src/Microsoft.HttpRepl.Tests/Commands/DeleteCommandsTest.cs

```diff
@@ -0,0 +1,110 @@
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.HttpRepl.Commands;
+using Microsoft.HttpRepl.Tests.SampleApi;
+using Microsoft.Repl;
+using Microsoft.Repl.ConsoleHandling;
+using Microsoft.Repl.Parsing;
+using Xunit;
+
+namespace Microsoft.HttpRepl.Tests.Commands
+{
+    public class DeleteCommandTests : IClassFixture<DeleteCommandsFixture>
+    {
+        private readonly DeleteCommandsConfig _config;
+        public DeleteCommandTests(DeleteCommandsFixture DeleteCommandsFixture)
+        {
+            _config = DeleteCommandsFixture.Config;
+        }
+
+        [Fact]
+        public async Task ExecuteAsync_WithNoBasePath_VerifyError()
+        {
+            HttpState httpState = new HttpState();
```

Comment:
Hey Tim, the tests here and the ones added before, seem to either verify error/outputs. Except the commandText and command itself, they all pretty much look the same. Do you think we can create a base class and put the Verify(..) methods there? That way we won't be duplicating the code for the other tests.

Reply:
Yeah, I initially thought they would be more different but now that I've added some more, it needs a refractor. I'll give it a shot.

Reply:
This has been pretty heavily refactored now. Take a look and see what you think.

Reply:
It looks good to me. Thank you!